AndroidTest/app/build.gradle

```diff
@@ -76,24 +64,33 @@ dependencies {
     // since our test harness is a standard android app which will run our tests,
     // we include the dependencies necessary for build and test here
 
+
+
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.1.1'
     compile group: 'com.google.guava', name: 'guava', version: '15.0'
     compile group: 'commons-io', name: 'commons-io', version: '2.4'
     compile group: 'commons-codec', name: 'commons-codec', version: '1.9'
-    compile group: 'org.hamcrest', name: 'hamcrest-library', version: '1.3'
-    compile group: 'junit', name: 'junit', version: '4.11'
-    compile group: 'org.mockito', name: 'mockito-core', version: '1.9.5'
+    androidTestCompile('org.hamcrest:hamcrest-library:1.3'){
+        force true
```

Comment:
when I was developing I was trying to fix multiple versions of junit and hamcrest being dexed, but fixed now in 69ea2c8